code401challenges/src/test/java/code401challenges/utilities/AnimalShelterTest.java

```diff
@@ -0,0 +1,25 @@
+package code401challenges.utilities;
+
+import code401challenges.stacksandqueues.Queue;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AnimalShelterTest {
+
+    @Test
+    public void testEnqueueAnimalShelter() {
+        AnimalShelter testShelter = new AnimalShelter();
+        Queue dog = new Queue<>();
+
+        testShelter.enqueue(dog);

```

Comment:
Since `dog` is a `queueu` these two lines are enqueue-ing queues into the animal shelter